docs/libs/fs.rst

```diff
@@ -14,6 +14,7 @@ LVG has built in support for:
 - POSIX (Linux and Windows using POSIX function .e.g ``open``, ``read``)
 - WIN32 (Windows using Win32 API function .e.g ``CreateFileA``, ``ReadFile``)
 - MEMFS (read a file from a memory buffer)
+- LITTLEFS (on-board file system for your ESP32 & ESP8266)
```

Comment:
as I know LittleFS is platform independent, so you can remove "ESP32 & ESP8266", or replace to:
_A little fail-safe filesystem designed for microcontrollers._

(first line of https://github.com/littlefs-project/littlefs/blob/master/README.md )